src/Entity/ProcurationProxy.php

```diff
@@ -287,6 +287,26 @@ class ProcurationProxy
      */
     public $recaptcha = '';
 
+    /**
+     * @var int
+     *
+     * @Assert\Range(
+     *     min=1,
+     *     max=3,
+     *     minMessage="procuration.vote_country.conditions",
+     *     maxMessage="procuration.vote_country.conditions",
+     *     groups={"front"}
+     * )
+     * @Assert\Expression(
+     *     "(this.getVoteCountry() == 'FR' and value <= 2) or (this.getVoteCountry() != 'FR' and value <= 3)",
+     *     message="procuration.vote_country.conditions",
+     *     groups={"front"}
+     * )
+     *
+     * @ORM\Column(type="integer",  options={"unsigned": true, "default": 1})
+     */
+    public $nbOfProxiesProposed = 1;
```

Comment:
what about `(total|count)ProposedProxies` or just `(total|count)Proxies`?

Reply:
`proxies_count` it is